src/backend/utils/sanitize-html.js

```diff
@@ -36,7 +40,6 @@ module.exports = function(dirty) {
       'h6',
       'thread',
       'caption',
-    ]),
-    disallowedTagsMode: 'discard',
```

Comment:
You removed this, but I think it's probably a good idea.  Do we not need it to remove tags that aren't listed here, or is this the default?